pkg/cluster/cloudstack_test.go

```diff
@@ -15,3 +20,64 @@ func TestParseConfigMissingCloudstackDatacenter(t *testing.T) {
 	g.Expect(err).To(Not(HaveOccurred()))
 	g.Expect(got.CloudStackDatacenter).To(BeNil())
 }
+
+func TestDefaultConfigClientBuilderBuildCloudStackClusterSuccess(t *testing.T) {
+	g := NewWithT(t)
+	ctx := context.Background()
+	b := cluster.NewDefaultConfigClientBuilder()
+	cluster := cloudStackCluster()
+	datacenter := &anywherev1.CloudStackDatacenterConfig{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       anywherev1.CloudStackDatacenterKind,
+			APIVersion: anywherev1.GroupVersion.String(),
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "datacenter",
+			Namespace: "default",
+		},
+	}
+
+	client := fake.NewClientBuilder().WithObjects().Build()
+	_, err := b.Build(ctx, clientutil.NewKubeClient(client), cluster)
+	g.Expect(err).To(MatchError(ContainSubstring("building Config from a cluster client")))
```

Comment:
isn't this testing the same as `TestDefaultConfigClientBuilderBuildCloudStackClusterFailure`

Reply:
Ah, apologize. This was suppose to be removed. Thanks for catching that!